Sources/DuetBot/main.swift

```diff
@@ -1,15 +1,27 @@
+import Foundation
 import Sword
 
-let bot = Sword(token: "token")
-
-bot.editStatus(to: "online", playing: "with Sword!")
-
-bot.on(.messageCreate) { data in
-  let msg = data as! Message
-
-  if msg.content == "!ping" {
-    msg.reply(with: "Pong!")
-  }
+guard let config = FileManager.default
+    .contents(atPath: FileManager.default.currentDirectoryPath.appending("/config.json"))
```

Comment:
Config should probably be in root instead of nested in directories

Reply:
If you're running this with swiftc I think this should just work; using Xcode you can put it wherever but you'd need to change the copy phase

Reply:
Also please add a config.json.example with an example structure